valhalla/jawn/src/managers/request/RequestManager.ts

```diff
@@ -168,4 +178,34 @@ export class RequestManager extends BaseManager {
           sort
         );
   }
+
+  async getAsset(
+    requestId: string,
+    assetId: string
+  ): Promise<Result<HeliconeRequestAsset, string>> {
+    const query = `
```

Comment:
All Db interactions belong in a store. 

This method would be something like:

getRequestAssetById